packages/esm-patient-medications-app/src/order-basket/order-basket-item-list.component.tsx

```diff
@@ -85,8 +88,9 @@ export default function OrderBasketItemList({
       {revisedOrderBasketItems.length > 0 && (
         <>
           <h4 className={styles.orderCategoryHeading}>
-            {t('revisedOrders', '{count} order(s) being modified (revised)', {
+            {t('revisedOrders', '{count} order{plural} being modified (revised)', {
               count: revisedOrderBasketItems.length,
+              plural: revisedOrderBasketItems.length > 1 ? 's' : '',
```

Comment:
It's probably better to use [i18next's plural support](https://www.i18next.com/translation-function/plurals) since just adding a "s" on the end is kind of a special case even in English.